styles.css

```diff
@@ -87,3 +91,401 @@ body {
+  left: 16px;
+  top: 284px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.yes {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.fine {
+  width: 236px;
+  height: 40px;
+  top: 200px;
+  left: 16px;
+  gap: 12px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+}
+
+.action {
+  width: 327px;
+  height: 48px;
+  left: 0;
+  top: 10px;
+  background: #ff6b00;
+  display: inline-block;
+}
+
+.try {
+  display: inline-block;
+  bottom: 0;
+  margin: 0;
+}
+
+#grid {
+  display: grid;
+  top: 1000px;
+  grid-row-gap: 60px;
+}
+
+#stories {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: 'normal';
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+#caption_img {
+  width: 327px;
+  height: 237px;
+  left: 24px;
+}
+
+#about_section {
+  background-image: url(/Image/about_me.png);
+  background-repeat: no-repeat;
+  background-size: 327px;
+  top: 4120px;
+  margin: 10px;
+  display: block;
+  left: 0;
+}
+
+#ment {
+  font-family: 'Crete Round', sans-serif;
+  font-style: 'normal';
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 88px;
+  letter-spacing: 0.36;
+  color: #ff6b00;
+  left: 24px;
+  margin: 24px;
+}
+
+#about {
+  width: 280px;
+  height: 148px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: white;
+  margin: 24px;
+  word-wrap: break-word;
+}
+
+#about_button {
+  width: 161px;
+  height: 48px;
+  gap: 10px;
+  left: 24px;
+  top: 395px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  margin: 24px;
+}
+
+#bar {
+  width: 297px;
+  height: 40px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+}
+
+#button {
+  width: 124px;
+  height: 48px;
+  color: white;
+  background-color: #ff6b00;
+  border: 0;
+  transition-duration: 0.6s;
+  padding: 12px;
+  top: 259px;
+}
+
+#button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  padding: 12px;
+  color: #f55800;
+}
+
+#button:active {
+  background: #e05304;
+  color: #f4f1f0;
+  width: 124px;
+  height: 48px;
+}
+
+#button.disabled {
+  opacity: 0.6;
+  cursor: not-allowed;
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+#button:enabled {
+  background: #f55800;
+  color: #fff;
+  width: 124px;
+  height: 48px;
+}
+
+.buttons {
+  width: 327px;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background: #ff6b00;
+}
+
+#skill {
+  width: 280px;
+  height: 1350px;
+  top: 4590px;
+  padding: 120px 24px 120px 24px;
+  background-color: #1c1a19;
+  left: 10px;
+  color: #fff;
+}
```

Comment:
Great work @Solobarine 👏

- [ ] [Optional] Please make sure you remove words that appear to be on top of each other.
![frameworks](https://user-images.githubusercontent.com/63917725/177011246-0406e315-31f4-4502-a41c-133e9fb145ee.PNG)

Reply:
@1GORDON  Thanks for the review. The first error you pointed out belonged to Wednesdays project but I corrected it.